Car/TeamNull SmartCar/src/util/HardwareUtil.cpp

```diff
@@ -58,58 +124,62 @@ void initialiseSensors() {
   // Serial.println("); or another similar value that works better according to your experimentation.");
   GY50 gyro(12);
 
-  // pinMode(rearSensorPin, OUTPUT);
-  // pinMode(frontSensorPin, OUTPUT);
-  // digitalWrite(rearSensorPin, LOW);
-  // digitalWrite(frontSensorPin, LOW);
+  pinMode(rearSensorPin, OUTPUT);
+  pinMode(frontSensorPin, OUTPUT);
+  digitalWrite(rearSensorPin, LOW);
+  digitalWrite(frontSensorPin, LOW);
 
-  // delay(1000);
+  delay(1000);
 
-  // //Set front sensor address
-  // // digitalWrite(frontSensorPin, HIGH);
-  // pinMode(frontSensorPin, INPUT);
-  // delay(150);
+  //Set front sensor address
+  // digitalWrite(frontSensorPin, HIGH);
+  pinMode(frontSensorPin, INPUT);
+  delay(150);
   // Serial.println("00");
   frontSensor.setTimeout(500);
   while (!frontSensor.init()) {
     Serial.println("Failed to initialise FRONT sensor");
     blink(2);
-    // car.set
-    // pinMode(frontSensorPin, OUTPUT);
-    // digitalWrite(frontSensorPin, LOW);
 
-    // delay(1500);
-    // pinMode(frontSensorPin, INPUT);
+    pinMode(frontSensorPin, OUTPUT);
+    digitalWrite(frontSensorPin, LOW);
+
+    delay(1500);
+    pinMode(frontSensorPin, INPUT);
   }
 
   delay(100);
-  // frontSensor.setAddress((uint8_t)01);
-
-  //Set rear sensor address
+  frontSensor.setAddress((uint8_t)01);
+  
   // Serial.println("01");
+  //Set rear sensor address
   // digitalWrite(rearSensorPin, HIGH);
   pinMode(rearSensorPin, INPUT);
   delay(150);
   // rearSensor.setTimeout(500);
   // // rearSensor.setAddress((uint8_t)01);
-  // while (!rearSensor.init()) {
-  //   Serial.println("Failed to initialise REAR sensor");
-  //   blink(2);
-  //   pinMode(frontSensorPin, OUTPUT);
-  //   digitalWrite(frontSensorPin, LOW);
-
-  //   delay(1500);
-  //   pinMode(frontSensorPin, INPUT);
-  // }
+  while (!rearSensor.init()) {
+    Serial.println("Failed to initialise REAR sensor");
+    blink(2);
+    pinMode(rearSensorPin, OUTPUT);
+    digitalWrite(rearSensorPin, LOW);
+
+    delay(1500);
+    pinMode(rearSensorPin, INPUT);
+  }
   // rearSensor.setAddress((uint8_t)02);
 
+  SR04 frontRight(FR_TRIGGER_PIN, FR_ECHO_PIN, MAX_DISTANCE);
+  SR04 frontLeft(FL_TRIGGER_PIN, FL_ECHO_PIN, MAX_DISTANCE);
```

Comment:
Why are you redeclaring these?

Reply:
That would be PEBCAK :/ resolved (in my branch)